src/shape/custom_shapes.js

```diff
@@ -2098,21 +2108,84 @@ function customShapes(p5, fn) {
    *   // Set the curve's tightness using the mouse.
    *   let t = map(mouseX, 0, 100, -5, 5, true);
    *   splineProperty('tightness', t);
-   * 
+   *   // 'ends' defaults to INCLUDE; uncomment to see the effect:
+   *   // splineProperty('ends', EXCLUDE);
    *   // Draw the curve.
    *   noFill();
    *   beginShape();
-   *   splineVertex(10, 26);
-   *   splineVertex(10, 26);
-   *   splineVertex(83, 24);
-   *   splineVertex(83, 61);
-   *   splineVertex(25, 65);
-   *   splineVertex(25, 65);
+   *   splineVertex(60, 76);
```

Comment:
With the default INCLUDE mode, we don't need to double the first and last vertex, it'll go through them automatically, so we can take this and the last `splineVertex` line out!